### PR TITLE
Task-38526: Add external label for external activity commenter

### PR DESCRIPTION
**Pull request by ahmadmido**

### CODE REVIEW

File: component/core/src/main/java/org/exoplatform/social/core/service/LinkProvider.java [hunk 1]
```diff
@@ -452,4 +462,35 @@ public static String getBaseURLSocialUserRest() {
...
+      return null;
+    }
+  }
+
+  /**
+   * Gets the resource bundle label.
+   *
+   * @return the label
+   */
+  public static String getResourceBundleLabel(Locale locale, String label) {
```
Comment (azayati) [inline component/core/src/main/java/org/exoplatform/social/core/service/LinkProvider.java:492]: I think that there is an Utils method added for that ?

File: extension/war/src/main/webapp/groovy/social/webui/activity/commons/UIActivityCommentBox.gtmpl [hunk 2]
```diff
@@ -12,6 +12,7 @@
   def labelComment = _ctx.appRes("UIActivity.label.Comment");
   def labelCancel = _ctx.appRes("UIActivity.label.Cancel");
   def labelUpdate = _ctx.appRes("UIActivity.label.Update");
+  def external = _ctx.appRes("UIActivity.label.External");
```
Comment (azayati) [inline extension/war/src/main/webapp/groovy/social/webui/activity/commons/UIActivityCommentBox.gtmpl:15]: The () should be removed from this label, to be added directly when displaying

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: component/core/src/main/java/org/exoplatform/social/core/service/LinkProvider.java (hunk 1 above)
Comment (ahmadmido) [reply component/core/src/main/java/org/exoplatform/social/core/service/LinkProvider.java:492]: It can't be used in core component !

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**